test/features/step_definitions/sdk_steps.rb

```diff
@@ -1,35 +1,40 @@
-Given(/^I verify with (passport|identity_card|drivers_license)$/) do |document_type|
+require_relative '../helpers/i18n_helper.rb'
+
+i18n = I18nHelper.new(@driver)
+
+Given(/^I verify with (passport|identity_card|drivers_license) with (es)?$/) do |document_type, locale|
```

Comment:
to make solution more generic, you could change the parameter from ` with (es)?` to `(?:| with (.+)?)`, so that it looks like:
`Given(/^I verify with (passport|identity_card|drivers_license)(?:| with (.+)?)$/)`

Thanks to this we would be able to:
1. Use just `I verify with (doc)` without the need of `with <locale>` at the end (backwards compatibility)
2. Specify locale other than just `es`

Reply:
@stefaniacardenas